### PR TITLE
Seperate HTML and JS code

### PR DESCRIPTION
**Pull request by dorlinski**
[Problem] HTML and JS code are mixed in documentation
[Solution] Add label "HTML code" and "JS code" if code snippet contains both JS and HTML code.

Signed-off-by: Dominik Orliński <d.orlinski@samsung.com>

### CODE REVIEW

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm [hunk 1]
```diff
@@ -65,18 +65,20 @@ <h2><a id='manual-constructor'></a>How to Create page indicator</h2>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm]: Question. Don't we need `<p>HTML code:</p>` here?

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm [hunk 2]
```diff
@@ -53,6 +53,7 @@ <h2>Table of Contents</h2>
 <h2 id="manual-constructor">How to Create Section Changer</h2>
 
 <p>To add a section changer component to the application, use the following code:</p>
+<p>HTML code:</p>
 <pre class="prettyprint"><pre class="prettyprint lang-html" style="border:0px; margin:0px">
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Is this `<pre class="prettyprint"><pre class="prettyprint lang-html" style="border:0px; margin:0px">` intended? 2 `<pre>` tags are duplicated. Related with `&lt;/div&gt;</pre>` on line 76 and `</pre>` on line 78.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm (hunk 1 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm]: Yes, we need. I will fix it.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm (hunk 2 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm (hunk 1 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm [hunk 3]
```diff
@@ -182,7 +183,6 @@ <h2>Popup Methods</h2>
    {
       tau.closePopup();
    });
-&lt;/script&gt;
 </pre></pre>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm]: Would you please check this \</pre>\</pre>?
It looks like it should be \</pre> to me.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm [hunk 4]
```diff
@@ -222,36 +222,38 @@ <h2>Popup Events</h2>
...
+<p>JS code:</p>
+<pre class="prettyprint">
    /* Use popup events */
    var popup = document.getElementById(&quot;popup&quot;);
    popup.addEventListener(&quot;popupbeforecreate&quot;, function(ev)
    {
       /* Implement code for popupbeforecreate event */
    });
-&lt;/script&gt;
 </pre></pre>
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm]: Would you please check this \</pre>\</pre>?
It looks like it should be \</pre> to me.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm [hunk 5]
```diff
@@ -72,7 +73,11 @@ <h2 id="manual-constructor">How to Create Section Changer</h2>
...
    &lt;/div&gt;
-&lt;/div&gt;</pre><pre class="prettyprint lang-js" style="border:0px; margin:0px">(function()
+&lt;/div&gt;
+</pre>
+<p>JS code:</p>
+<pre class="prettyprint">
+(function()
 {
    var page = document.getElementById(&quot;hasSectionchangerPage&quot;),
        element = document.getElementById(&quot;sectionchanger&quot;),
```
Comment (mijin85cho) [inline docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Would you please check this \</pre>\</pre> on line 103?
It looks like it should be \</pre> to me.

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm (hunk 3 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm (hunk 4 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_popup.htm]: Fixed

File: docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm (hunk 5 above)
Comment (dorlinski) [reply docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm]: Fixed

**Comment by dorlinski** [comment]
@TomaszLukawskiSam please review

**Pull request merged**